tools/test/scripts/nb_test/set.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+ONOS_CONFIG_ROOT=$HOME/go/src/github.com/onosproject/onos-config
+
+gnmi_cli -address localhost:5150 -set \
+    -proto "update: <path: <target: 'localhost:10161', elem: <name: 'system'> elem: <name: 'clock' > elem: <name: 'config'> elem: <name: 'timezone-name'>> val: <string_val: 'Europe/Dublin'>>" \
```

Comment:
the problem is the proto - were you going to hard code it like this? It will always update the timezone, and so maybe the script should have that as a name

Reply:
I was thinking to define it as a variable on top of the script that can be changed easily. I assume that the -proto argument can be changed by the user. Is that what you are suggesting?

Reply:
@SeanCondon 
I made the changes and introduced some variables to make the scripts more general. I couldn't test subscribe because it doesn't work and we need to take a look at it later.

Reply:
I know I might be a bit of a pain here. I would not extract these in other commands. gnome_cli it's an external tool from ONOS-config so I woudl not tie us to it in any way. Also to me the cli is a pure testing tool so c/p is not an issue and actually enables me to learn more about what's going on If I see the whole command